cybershell/fingerprint.py

```
@@ -0,0 +1,649 @@
+                        self._analyze_endpoint_result(fingerprint, endpoint, result)
+                except:
+                    pass
+    
+    def _check_endpoint(self, url: str) -> Optional[Dict]:
+        """Check if endpoint exists and return info"""
+        try:
+            response = requests.get(
+                url,
+                timeout=5,
+                headers={'User-Agent': 'Mozilla/5.0'},
+                verify=False,
```

Ruff (0.12.2)

Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)